sms-broadcast/README.md

```diff
@@ -0,0 +1,86 @@
+
+### Create a Messaging Service
+
+Go to the console and [create a messaging service](https://www.twilio.com/console/sms/services). Choose the "Notifications, Two-way" use case to preset some options on the service.
+
+Once created, click on the left nav link for "Numbers" and add (or buy) a phone number to use with this service. This will be the phone number(s) your end users will interact with.
+
+We'll need to come back here later - consider leaving this page open in a tab while you proceed to the next step in the console.
+
+### Create a Notify Service
+
+In the console, create a [Notify service](https://www.twilio.com/console/notify/services). Our function code uses Notify to store our subscriber list and to send out notifications. After your service is created, locate the Dropdown in your Notify service config labeled "Messaging Service SID". You should be able to choose the Messaging Service we created in the last step.
+
+You will need the generated SID for this service to configure your environment in the next step.
+
+Don't forget to save your changes!
+
+### Deploy the Function template
+
+Add the Notify Service SID to the `.env` file as `BROADCAST_NOTIFY_SERVICE_SID`. Add the phone numbers of admins that are permitted to send broadcast messages to the `.env` file as `BROADCAST_ADMIN_NUMBERS`.
+
+Deploy your functions and assets with the following command. Note: you must run this command from inside your project folder. [More details in the docs.](https://www.twilio.com/docs/labs/serverless-toolkit)
+
+To deploy the function with the [Twilio CLI](https://www.twilio.com/docs/twilio-cli/quickstart) run the following:
+
+```
+twilio serverless:deploy
+```
+
+In a few moments your Function should be deployed! Grab its URL from the results, as we will now need to configure it for incoming SMS messages to our Messaging service.
+
+### Handle incoming messages with your Function
+
+Go back to the [messaging service](https://www.twilio.com/console/sms/services) you created earlier. In its configuration, you'll see a checkbox for "Handle Incoming Messages". Click this box to enable the feature. In the text box that appears, paste in the URL to the Function we just deployed. Click "Save".
+
+Now if all went well, you'll be able to start using your phone number for managing broadcasts and subscriptions!
+
+## Usage
+
+Once the Function is deployed and your Twilio phone number is set up, folks can text anything to it to receive a brief informational message about what commands are available.
+
+* `subscribe` - subscribes the current number for updates from the service
+* `stop` - uses Twilio's built-in stop handling to prevent a user from receiving messages
+* `start` - uses Twilio's built-in features to opt a user back in to receiving messages
```

Comment:
Similarly, a brief sentence here describing the mechanism of action (sending number is removed from the blocklist if it's present) might be warranted.